lib/oauth/request_proxy/action_controller_request.rb

```diff
@@ -72,6 +72,10 @@ def parameters_for_signature
         reject { |kv| kv[0] == 'oauth_signature'}
     end
 
+    def raw_post_signature
```

Comment:
IMO this should be `raw_post_signature?` with a `?` for convention (it returns a boolean). Minor nitpick though

Reply:
Ok, fixed!